src/content/quickstart/openshiftgn/_index.md

```diff
@@ -84,6 +84,9 @@ subctl join --kubecontext west --kubeconfig ./merged_kubeconfig broker-info.subm
 subctl join --kubecontext east --kubeconfig ./merged_kubeconfig broker-info.subm --clusterid east --broker-cluster-context west
 ```
 
+By default `subctl` will auto-assign global CIDRs to the clusters. The `--globalnet-cidr` flag can be used if the Globalnet CIDRs need to be assigned explicitly.
+subctl will check that the specified CIDRs do not overlap across clusters before joining the clusters.
```

Comment:
Since this is a quick start guide, IMO its better to keep this short and move any additional information about GN to the corresponding GN architecture page. Reason being, if you see, as part of deploy-broker command above, we are not including any notes about default --globalnet-cidr-range, --globalnet-cluster-size etc. In GN architecture page, we can add some notes about the various globalnet parameters and how they work.

Reply:
I have added few lins about globalnet-cdr in the architecture page as well.